# netsurf: misc
# 1 files, 2 changed lines
#
# Regenerate: tools/regen-patches.sh && tools/split-patches.py
#
diff -urN -x .git -x 'build-*' -x test -x frontends -x '*.o' -x '*.a' -x '*.d' -x '*.pc' -x build upstream/netsurf/resources/default.css browser/netsurf/resources/default.css
--- upstream/netsurf/resources/default.css	2026-09-20 23:29:00.170662942 +0100
+++ browser/netsurf/resources/default.css	2026-09-20 23:19:21.016806698 +0100
@@ -111,7 +111,7 @@
 
 form { display: block; }
 
-input, button { background-color: #fff; color: #000; text-align: left;
+input, button { display: inline-block; background-color: #fff; color: #000; text-align: left;
 	font-family: sans-serif; width: auto; height: auto; overflow: hidden;
 	border: 1px solid #444; padding: 2px 3px; line-height: 1.33;
 	margin: 1px; box-sizing: border-box; }
